api/src/backend/api/v1/serializers.py

```diff
@@ -1532,6 +1543,40 @@ class Meta:
         resource_name = "provider-secrets"
 
 
+class AlibabaCloudProviderSecret(serializers.Serializer):
+    access_key_id = serializers.CharField()
+    access_key_secret = serializers.CharField()
+    security_token = serializers.CharField(required=False)
+
+    class Meta:
+        resource_name = "provider-secrets"
+
+
+class AlibabaCloudRoleAssumptionProviderSecret(serializers.Serializer):
+    """Serializer for Alibaba Cloud RAM Role Assumption credentials.
+
+    This allows assuming a RAM role using access keys, similar to AWS STS AssumeRole.
+    The SDK will automatically manage the STS token refresh.
+    """
+
+    role_arn = serializers.CharField(
+        help_text="ARN of the RAM role to assume (e.g., acs:ram::1234567890123456:role/ProwlerRole)"
```

Comment:
This is to maintain consistency 

```suggestion
        help_text="Access Key ID of the RAM user that will assume the role"
```

Reply:
Done